projects/M4C/scripts/extract_ocr_frcn_feature.py

```diff
@@ -0,0 +1,193 @@
+import os
+import numpy as np
+import tqdm
+import argparse
+import torch
+from PIL import Image
+import cv2
+
+# install `vqa-maskrcnn-benchmark` from
+# https://github.com/ronghanghu/vqa-maskrcnn-benchmark-m4c
+import sys; sys.path.append('/private/home/ronghanghu/workspace/vqa-maskrcnn-benchmark')  # NoQA
```

Comment:
Is there some specific change done for m4c? If not can the `vqa-maskrcnn-benchmark` repo be used here?  Because it is already being used in feature extraction scripts pythia provides here `pythia/scripts/features/extract_features_vmb.py`

Reply:
Yeah, this needs a separate look.

Reply:
Yes, there are specific changes for OCR feature extraction. The major change is to allow RoI-Pooling from an externally-specified bounding box (OCR boxes in our use case) instead of from the Faster R-CNN's own RPN proposal. The new branch (https://github.com/ronghanghu/vqa-maskrcnn-benchmark-m4c) is compatible with pythia/scripts/features/extract_features_vmb.py, so can also land it into https://gitlab.com/meetshah1995/vqa-maskrcnn-benchmark.